lib/app/components/pages/dashboard/views/dashboard_view.dart

```diff
@@ -44,25 +40,17 @@ class DashboardView extends GetView<DashboardController> {
       ),
       bottomNavigationBar: Obx(
         () => BottomNavigationBar(
-          onTap: controller.tabService.changeIndex,
+          onTap: controller.changeIndex,
           currentIndex: controller.tabService.tabIndex.value,
           unselectedItemColor: colorDashboardUnselectedItem,
           selectedItemColor: colorDashboardSelectedItem,
           type: BottomNavigationBarType.fixed,
-          items: const <BottomNavigationBarItem>[
+          items: controller.navigationInfo.map((info) =>
```

Comment:
```suggestion
          items: NavigationUtil.bottomBarItems
```
`NavigationUtil`クラスにまとめちゃったのでこのようにしましょう

Reply:
```suggestion
          items: bottomBarItems
```
上のように修正お願いします。
`organisms`にファイル作ったらimportもしてくださいね。